be/tool/beck.c

```diff
@@ -1554,16 +1624,37 @@ static void builder_fini(struct builder *b)
 {
 	m0_thread_join(&b->b_thread);
 	m0_thread_fini(&b->b_thread);
-	qfini(&b->b_qq);
 	m0_ctg_store_fini();
 	m0_reqh_be_fini(&b->b_reqh);
 	ad_dom_fini(b);
 	m0_be_ut_backend_fini(&b->b_backend);
 	m0_reqh_fini(&b->b_reqh);
 	m0_free(b->b_backend.but_stob_domain_location);
 
-	printf("builder: actions: %9"PRId64" txs: %9"PRId64"\n",
-	       b->b_act, b->b_tx);
+	printf("builder: actions: %9"PRId64"\n", b->b_act);
+}
+
+static void  be_cfg_default_init(struct m0_be_domain_cfg  *dom_cfg,
+				 struct m0_be_tx_bulk_cfg *tb_cfg)
```

Comment:
Mention some comment indicating these values provided maximum performance after experimentation. Or else Code reader will not know why these values were used instead of the ones which are filled by default for be_cfg.

Reply:
Done.